definitions/pipelines/gathered_somatic_exome.cwl

```diff
@@ -129,6 +129,8 @@ inputs:
         secondaryFiles: [.tbi]
     output_dir: 
         type: string
+    somalier_vcf:
```

Comment:
maybe "concordance_vcf", since that's more generally descriptive than somalier is?